src/zope/meta/config_package.py

```diff
@@ -197,6 +205,21 @@ def config_type(self):
                 'Please use `--type` to select it.')
         return value
 
+    @cached_property
+    def oldest_python(self):
+        value = (self.meta_cfg['python'].get('oldest-python') or
+                 self.args.oldest_python)
```

Comment:
I assume that `self.args` refers to the command line arguments. Usually, those take precedence over other configuration sources.

Reply:
You're right, I changed the ordering.